packages/app/config/babel.prod.js

```diff
@@ -4,33 +4,32 @@ module.exports = {
   // Don't try to find .babelrc because we want to force this configuration.
   babelrc: false,
   presets: [
+    require.resolve('@babel/preset-flow'),
     // Latest stable ECMAScript features
     [
       'env',
       {
         targets: {
           ie: 11,
-          // We currently minify with uglify
-          // Remove after https://github.com/mishoo/UglifyJS2/issues/448
-          uglify: true,
         },
         // Disable polyfill transforms
         useBuiltIns: false,
         modules: false,
+        forceAllTransforms: true,
```

Comment:
What does this mean?

Reply:
This is a replacement for the `uglify` option that used to be there. They probably wanted to name the option what it actually does, not what like "I have uglify" type of option.